core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionExecutionCoordinator.scala

```diff
@@ -150,23 +149,40 @@ class RegionExecutionCoordinator(
           val inputPortMapping = physicalOp.inputPorts
             .flatMap {
               case (inputPortId, (_, _, Right(schema))) =>
-                Some(GlobalPortIdentity(physicalOp.id, inputPortId, input = true) -> schema)
+                Some(GlobalPortIdentity(physicalOp.id, inputPortId, input = true) -> ("", schema))
```

Comment:
why are we changing for input ports? I thought storage only affects output ports.

Reply:
This is needed because `AssignPortRequest` is for both input and output ports. Please see this complete code block for the expected output of the flapMap.

Reply:
understood that the assign port request is written in a way that works for both input and output ports. This is because previously, the input and output ports had the same amount of information. Now as we are adding new information to the output port (stroage URI). it makes more sense to change the implementation such that input and output ports are handled differently. 

However, recall that we also need may storage URI for inputs in the future (for replacing cache read operator). Currently in this PR, a storage URI assigned to input has no effect. 
So you could assign an empty string storage URI to input ports, but can you at least add documentation and TODO to explain the difference with output ports, for now?

Reply:
Added the documentation and TODO.